### PR TITLE
feat: implements changes to support safe mode during authenticator en…

### PR DESCRIPTION
**Pull request by gowthamidommety-okta**
…rollment

Resolves: OKTA-345300

## Description:

See https://github.com/okta/okta-signin-widget/pull/1529 for approval

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-345300](https://oktainc.atlassian.net/browse/OKTA-345300)

### CODE REVIEW

**Review by magizh-okta — APPROVED** [review]

File: src/v2/view-builder/utils/LinksUtil.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ const getSkipSetupLink = (appState) => {
         'type': 'link',
         'label': loc('oie.enroll.skip.setup', 'login'),
         'name': 'skip-setup',
-        'formName': RemediationForms.SKIP,
+        'actionPath': RemediationForms.SKIP,
```
Comment (mauriciocastillosilva-okta) [inline src/v2/view-builder/utils/LinksUtil.js:102]: Why is this not a `formName`? From the response I can see that it is inside the remediation object...

I can see that we don't want this to submit a form, but instead just follow the link. So I am guessing that is the reason, which makes sense. With that said, then I think we should not have `RemediationForms` constants assigned to an `actionPath`, those should probably be used only for `formNames`
I see we already have a mix in our codebase though... so as a nit, maybe we change it to:
```
'actionPath': 'skip',
```
which is what we already have in [ReEnrollAuthenticatorWarningPasswordView](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js#L46)

**Review by mauriciocastillosilva-okta — APPROVED** [review]
just one nit, rest looks good

File: src/v2/view-builder/utils/LinksUtil.js (hunk 1 above)
Comment (gowthamidommety-okta) [reply src/v2/view-builder/utils/LinksUtil.js:102]: Yea i was thinking about this too.. I thought the action path needs to be the form name (https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.js#L113), so I set it to this constant. Will change 👍

Comment (gowthamidommety-okta) [reply src/v2/view-builder/utils/LinksUtil.js:102]: Talked to Mau, leaving as is for now.. Will discuss with the team later

Comment (haishengwu-okta) [reply src/v2/view-builder/utils/LinksUtil.js:102]: It shall be `actionPath`.
- `actionPath` :  find the action from remediation list or other top level object and submit request
- `formName` : re-render page using this *remedation* form.

**Pull request closed without merging**